provider/go.mod

```diff
@@ -2,6 +2,11 @@ module github.com/pulumi/pulumi-kubernetes/provider/v4
 
 go 1.21
 
+replace (
```

Comment:
They're not for local testing, it is because the component provider uses the Go SDK. There's a similar block in [tests/go.mod](https://github.com/pulumi/pulumi-kubernetes/blob/v4.9.0/tests/go.mod#L5-L8).